scripts/cc-cli/src/commands/rotateKeys.ts

```diff
@@ -0,0 +1,17 @@
+import { Command, OptionValues } from "commander";
+import { newApi } from "../api";
+
+export function makeRotateKeysCommand() {
+  const cmd = new Command("rotate-keys");
+  cmd.option("-u, --url [url]", "URL for the Substrate node");
+  cmd.description("Rotate session keys for a specified node");
+  cmd.action(rotateKeysAction);
+  return cmd;
+}
+
+async function rotateKeysAction(options: OptionValues) {
+  const api = await newApi(options.url);
+  const newKeys = await api.rpc.author.rotateKeys();
+  console.log(newKeys.toString());
```

Comment:
```suggestion
  console.log("New keys: " + newKeys.toString());
```
Maybe a bit clearer than just seeing some hex strings printed